src/test/java/bio/terra/service/dataset/DatasetDaoTest.java

```diff
@@ -524,23 +522,28 @@ public void primaryKeyTest() throws Exception {
 
     assertThat(
         "single-column primary keys are set correctly",
-        variants.getPrimaryKey().stream().map(Column::getName).collect(Collectors.toList()),
-        equalTo(Collections.singletonList("id")));
+        variants.getPrimaryKey(),
+        containsInAnyOrder(variants.getColumnByName("id").get()));
 
     assertThat(
         "dual-column primary keys are set correctly",
-        metaAnalysis.getPrimaryKey().stream().map(Column::getName).collect(Collectors.toList()),
-        equalTo(Arrays.asList("variant_id", "phenotype")));
+        metaAnalysis.getPrimaryKey(),
+        containsInAnyOrder(
+            metaAnalysis.getColumnByName("variant_id").get(),
```

Comment:
These should probably be `getOrElseThrow()` instead of `get()`. 
```suggestion
            metaAnalysis.getColumnByName("variant_id").getOrElseThrow(),
```

Reply:
I don't think that exists...

Reply:
Oh wait, it's `orElseThrow`, nvm